components/ConvertForm/ManageConversion.js

```diff
@@ -1,20 +1,37 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useCallback } from 'react'
 import {
   useOpenOrder,
   useClaimOrder,
   useApprove,
   useAllowance,
+  useTransactonReceipt,
 } from 'lib/web3-contracts'
 import { bigNum } from 'lib/utils'
 import ConvertSteps from 'components/ConvertSteps/ConvertSteps'
 import processing from './assets/loader.gif'
 
-function ManageConversion({ toAnj, fromAmount, toAmount, handleReturnHome }) {
+function ManageConversion({ toAnj, fromAmount, handleReturnHome }) {
   const openOrder = useOpenOrder()
   const claimOrder = useClaimOrder()
+  const transactionReceipt = useTransactonReceipt()
   const changeAllowance = useApprove()
   const getAllowance = useAllowance()
   const [conversionSteps, setConversionSteps] = useState()
```

Comment:
This is more related to the previous PR, but I wonder if this should default to `[]`?